src/new-client/src/App/Trades/TradesGrid/components/ComparatorSelect.tsx

```diff
@@ -23,78 +20,39 @@ export const ComparatorSelectOuter = styled.div`
   border-bottom: 1px solid ${colors.spectrum.blue.base};
 `
 
-const ComparatorSelectInner = styled.div<{ visible: boolean }>`
-  border-radius: 4px;
-  display: ${({ visible }) => (visible ? "block" : "none")};
-  position: absolute;
-  top: 40px;
-  left: 0px;
-  background-color: ${({ theme }) => theme.primary.base};
-  padding: 6px;
-  box-shadow: 0 7px 26px 0 rgba(23, 24, 25, 0.5);
-  z-index: 100;
-  width: 100%;
-`
-
-const ComparatorOption = styled.div`
-  padding: 8px 8px 5px 8px;
-  font-weight: normal;
-  background-color: inherit;
-  border-radius: 2px;
-  margin-bottom: 5px;
-
+const ComparatorSelectInner = styled.select`
+  color: white;
```

Comment:
Need to handle light theme too.  Trying using a theme variable.

Reply:
yes that always get me lol. fixed

> Need to handle light theme too. Trying using a theme variable.